### PR TITLE
AQ-44 React Router setup

### PR DESCRIPTION
**Pull request by ToJestKrzysio**
# Description
Added React Router v6.9
Created `ProtectedRoute` Component, which redirects user to login if user has no valid access / refresh token.
Updated AuthToken adding `login`, `isTokenValid`, `isRefreshTokenValid`.
Added coverage.

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] The related memme has been added
image

### CODE REVIEW

File: quiz/web/src/App.tsx [hunk 1]
```diff
@@ -1,14 +1,31 @@
 import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
 import { AuthConfig, AuthTokenProvider } from "modules/common/AuthToken";
+import ErrorBoundary from "modules/common/components/ErrorBoundary";
+import { BrowserRouter, createBrowserRouter, RouterProvider } from "react-router-dom";
+
+const router = createBrowserRouter([
```
Comment (pawelkonior) [inline quiz/web/src/App.tsx:6]: I think that will be better if we create separate file for the router. Might be big in the future.

**Review by pawelkonior — COMMENTED** [review]
Looks good. One small fix.

File: quiz/web/src/App.tsx (hunk 1 above)
Comment (ToJestKrzysio) [reply quiz/web/src/App.tsx]: fixed

**Review by pawelkonior — APPROVED** [review]
Impressive

**Review by olivergav — APPROVED** [review]

**Pull request merged**